pkg/arguments/interactive.go

```diff
@@ -280,12 +281,31 @@ func PromptFilePath(fs *pflag.FlagSet, flagName string, required bool) error {
 			if err != nil {
 				return err
 			}
+			response, err = resolveRelativePath(response)
+			if err != nil {
+				return err
+			}
 			fs.Set(flagName, response)
 		}
 		return nil
 	})
 }
 
+// Golang does not support tilde file paths https://github.com/golang/go/issues/57569
+// However, we try to resolve this by manually so user can proceed further
+func resolveRelativePath(path string) (string, error) {
+	usr, err := user.Current()
```

Comment:
The majority of calls won't have the ~.

We could add a check and return earlier

Reply:
Added